agent/qrexec-agent-data.c

```diff
@@ -281,9 +281,9 @@ static int handle_new_process_common(
     req.prefix_data.data = NULL;
     req.prefix_data.len = 0;
 
-    exit_code = process_io(&req);
+    exit_code = qrexec_process_io(&req, cmd);
 
-    if (type == MSG_EXEC_CMDLINE)
+    if (type == MSG_EXEC_CMDLINE && pid > 0)
         LOG(INFO, "pid %d exited with %d", pid, exit_code);
```

Comment:
Can I get some log message on service exit?

Reply:
I think that should be a separate commit, since the current log message is completely unhelpful.

Reply:
It is helpful in that journald adds the logging process PID to the message, which can be easily correlated with the service start log.